src/Business Foundation/App/NoSeries/src/Setup/NoSeriesType.Enum.al

```diff
@@ -0,0 +1,17 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+enum 396 "No. Series Type"
+{
+    Access = Public;
+    Extensible = true;
+    
```

Comment:
Add comment block, with summary e.g.:
```suggestion

/// <summary>
/// A default type, specifiying that the No. Series is not customized.
/// </summary>

```

Reply:
This type is still WIP and is likely to be removed from here. It's currently used to support IT functionality.

Reply:
We will obviously add docs if it's kept!